common/lib/capa/capa/templates/crystallography.html

```diff
@@ -16,13 +17,13 @@
     <input type="text" name="input_${id}" aria-describedby="answer_${id}" id="input_${id}" value="${value|h}" style="display:none;"/>
 
     <p class="status" aria-describedby="input_${id}">
-        ${status.display_name}
+        <span class="sr">${status.display_name}</span>
     </p>
 
     <p id="answer_${id}" class="answer"></p>
 
     % if msg:
-        <span class="message">${msg|n}</span>
+        <span class="message" aria-describedby="answer_input_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
Not sure how much it matters given that we know nothing about crystallography, but "answer_input_${id}" does not ext anyplace else in this HTML, and thus is most likely not a valid value for aria-describedby.